crates/xilem_core/src/view/mod.rs

```diff
@@ -36,9 +37,9 @@ macro_rules! generate_view_trait {
         /// and also a type for actions which are passed up the tree in message
         /// propagation. During message handling, mutable access to the app state is
         /// given to view nodes, which in turn can expose it to callbacks.
-        pub trait $viewtrait<T, A = ()> $( $ss )* {
+        pub trait $viewtrait<T, A = ()> $( $super_bounds )* {
```

Comment:
Same here with `:`, I'm pretty sure `trait Foo<T, A = ()>: {}` is a valid trait definition with an empty list of supertraits.